<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>271</width>
      <height>491</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="2">
      <widget class="QPushButton" name="clear_values">
       <property name="text">
        <string>Clear Values</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="Current_club">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Club</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLineEdit" name="Goals">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Goals</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLineEdit" name="appearences">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Appearances</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="assists">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Assists</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="add_players">
       <property name="text">
        <string>Add Player</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QPushButton" name="update_stats">
       <property name="text">
        <string>Update Stats</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="filter_nationality">
       <property name="text">
        <string>Filter by Nationality</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QPushButton" name="filter_position">
       <property name="text">
        <string>Filter by Position</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QPushButton" name="display_players">
       <property name="text">
        <string>Display All Players</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="delete_players">
       <property name="text">
        <string>Delete Player</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Player Name</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Position</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Nationality</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Club</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Goals</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Assists</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Appearances</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QComboBox" name="Position">
       <item>
        <property name="text">
         <string>Striker</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Winger</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Attacking Midfielder</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Defensive Midfielder</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Full Back</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Center Back</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Goalkeeper</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLineEdit" name="Player_name">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Player Name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Player Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLineEdit" name="Nationality">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Nationality</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>325</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="text">
     <string>Soccer Player Database</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
